applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -752,6 +752,15 @@ static void on_device_found(const bt_addr_le_t *addr, int8_t rssi, uint8_t type,
 static void ble_acl_start_scan(void)
 {
 	int ret;
+	/* Scan interval as two times of ISO interval = 20 ms (0x20 * 0.625) */
```

Comment:
The comments should not mention 20 ms as this may change. Suggest
```suggestion
	/* Scan interval as two times of ISO interval */
```